packages/ag-trade/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@endo/skel",
```

Comment:
looks like the skeleton unaltered

Reply:
yes, this is exactly the skeleton, unaltered; i.e. before we start playing

I'm a little surprised that there are any `ag-trade/` files here at all, but I suppose the way I did this was to run the skeleton widget and then try to test / lint and add equipment as needed